app/scripts/controllers/metametrics.js

```diff
@@ -330,7 +330,11 @@ export default class MetaMetricsController {
   async trackEvent(payload, options) {
     // event and category are required fields for all payloads
     if (!payload.event || !payload.category) {
-      throw new Error('Must specify event and category.');
+      throw new Error(
+        `Must specify event and category. Passed payload was: ${JSON.stringify(
+          payload,
```

Comment:
Do we track the user's MetaMetricsId in sentry context @Gudahtt? If so tracking the entire payload is problematic here for anonymity reasons. It's likely safe here to just include the event OR category and possibly an array of keys supplied in the payload versus the values.

Reply:
I had the same thought, but it looks like the payload doesn't include the MetaMetricsId. That's in the options.

Reply:
Oh but the `properties` and `sensitiveProperties` are _a lot_. Especially the `sensitiveProperties` - those explicitly should not be included for privacy reasons. Good point - I agree with your recommendation.

Reply:
its also in the *sentry* appState context. 
<img width="902" alt="Screen Shot 2021-05-24 at 12 29 34 PM" src="https://user-images.githubusercontent.com/4448075/119385550-e1d15300-bc8b-11eb-9fe0-5616bdf14c41.png">